internal/broker/bind_get.go

```diff
@@ -22,7 +27,27 @@ func (b *GetBindingEndpoint) GetBinding(_ context.Context, instanceID, bindingID
 	b.log.Infof("GetBinding instanceID: %s", instanceID)
 	b.log.Infof("GetBinding bindingID: %s", bindingID)
 
+	binding, err := b.bindings.Get(instanceID, bindingID)
+
+	if binding == nil {
+		message := "Binding not found"
+		return domain.GetBindingSpec{}, apiresponses.NewFailureResponse(fmt.Errorf(message), http.StatusNotFound, message)
+	}
+
+	if binding.InstanceID != instanceID {
```

Comment:
Is it really necessary to check if we received the same instanceID? Seems too defensive.

Reply:
I want to connect search with both bindingId and instanceId available in URL. Otherwise, it would be possible to load bindings from different instance than the one that instanceId refers to.

Reply:
Eventually I got it.